diff --git a/dist/base.js b/dist/base.js
index cfd60a665fb366ed153c464c0a1822771235d8dc..68fe044bd961f636f2b3c83b30c6edc107a3312e 100644
--- a/dist/base.js
+++ b/dist/base.js
@@ -31,7 +31,7 @@ export const restrictEnvAccess = tseslint.config({ ignores: ["**/{_,}env.{js,ts}
 });
 export default tseslint.config(
 /** ignore files not tracked by VCS + any config files */
-includeIgnoreFile(path.join(import.meta.dirname, "../../../../.gitignore")), { ignores: ["**/*.config.*"] }, {
+{ ignores: ["**/*.config.*"] }, {
     files: ["**/*.js", "**/*.ts", "**/*.tsx"],
     ignores: ["**/dist/**/*.js"],
     plugins: {
